### DISCUSSION TITLE
How to add multiple images in the title slide of a powerpoint presentation?

### DISCUSSION THREAD

**Discussion by udurraniAtPresage** [quarto-dev/quarto-cli#9563] [Q&A] [answered]
### Description

I understand that for revealjs I can use css to put images on the title slide. For powerpoint, I could also just put the images in the title slide of reference template.pptx. But I need to change the images depending on the client. Do I need to always edit the template.pptx or is there something I can do in the qmd file to add the images in the title slide?

**Comment by mcanouil** [comment, marked as answer]
I believe there are no other choices which is unfortunately not really up to Quarto as this is Pandoc behaviour.

```sh
quarto pandoc index.qmd --from markdown --to pptx -o presentation.pptx
```

````qmd
---
title: "![Title](https://placehold.co/60x40.png)"
author: "Your Name"
date: "Today"
---

# Slide 1

Hello, world!

# Slide 2

- Bullet 1
- Bullet 2
- Bullet 3
````

**Reply by mcanouil** [reply]
CC. @cderv in case I am missing something.

**Reply by cderv** [reply]
Yes this is Pandoc limitation. The layout choice is based on slide content as described in their MANUAL
https://pandoc.org/MANUAL.html#powerpoint-layout-choice

And for title slide, they fill it based on meta information, and it seems they consider image. 

> Do I need to always edit the template.pptx or is there something I can do in the qmd file to add the images in the title slide?

So Quarto nor Pandoc won't offer you something that helps. However, know that some other tools can help post process some office files, and maybe you can replace the image from title slide as a post processing change. 

An example of a R 📦 to do pptx processing : https://ardata-fr.github.io/officeverse/officer-for-powerpoint.html

If that helps doing it, you can then have a workflow of 
- quarto render your document
- run post processing step on this document

This is two steps, but could be automated.

**Reply by udurraniAtPresage** [reply]
Thanks for the suggestions.
